js/tiles.js

```diff
@@ -59,6 +59,8 @@ ImageCollection.prototype.showIn = function(container) {
 };
 ImageCollection.prototype.playerEnters = function(player) {
   this.container.classList.add("current");
+  /* 214.772 is half of the tile width while 128.157 is half the height of the tile */
+  player.setPosition(this.pixelPosition.x+214.772, this.pixelPosition.y+128.157);
```

Comment:
please do not use hard-coded values. There are variables for that which you should use.
For better understanding, you can add a method getCenterPixelPosition() and pass that to setPosition.

And I think, if you rename `setPosition` to `displayAtPosition`, this should be more descriptive.

Reply:
Calling the player back is nice. I like it. :)

Reply:
sure